recipes/sdl/all/conanfile.py

```diff
@@ -192,6 +192,11 @@ def _configure_cmake(self):
                 self._cmake.definitions["PULSEAUDIO"] = self.options.pulse
                 self._cmake.definitions["SNDIO"] = self.options.sndio
                 self._cmake.definitions["NAS"] = self.options.nas
+                if self.options.nas:
+                    libs = ["-l{}".format(name) for name in (self.deps_cpp_info["xorg"].libs + self.deps_cpp_info["xorg"].system_libs)]
+                    self._cmake.definitions["EXTRA_LDFLAGS"] = " ".join(libs)  # SDL is not populating transitive deps!
```

Comment:
🤷  The alternative is to patch/modify around the lines where the NAS library is discovered and _imported_... maybe better.

Reply:
I would hardcode `-lXau`, and add a fixme comment.
The xorg recipe is too big. The shared library will link to all xorg libraries instead of only `libXau.so`.

Reply:
It search for all of this... it can help if we _fix sdl build_system_ and want to depend only on those components:

```
    foreach(_LIB X11 Xext Xcursor Xinerama Xi Xrandr Xrender Xss Xxf86vm)
        FindLibraryAndSONAME("${_LIB}")
    endforeach()
```